public/js/components/AtomList/AtomList.js

```diff
@@ -0,0 +1,173 @@
+            }
+        }
+
+        return <a target="_blank"
+                  href={externalEditorUrl(this.props.config.atomEditorUrls)}
+                  className="atom-list__link atom-list__editor-link">
+                {title}</a>;
+    };
+
+    render () {
+
+        if (!this.props.atomList) {
+            return <div>Loading...</div>;
+        }
+
+        return (
+            <div className="page__section">
+                <h1 className="page__subheading">Atom Finder</h1>
+                <div className="atom-list-filters">
+                    <form className="form atom-list-filters">
+                        <div className="atom-list-filters__search-box">
+                            <ManagedField data={this.state.params} updateData={this.updateAtomList} fieldLocation="q" name="Search:">
+                                <FormFieldTextInput/>
+                            </ManagedField>
+                        </div>
+                        <div className="atom-list-filters__types-filter">
+                            <ManagedField data={this.state.params}
+                                          updateData={this.updateAtomList}
+                                          fieldLocation="types"
+                                          name="Atom Types:">
+                                <FormFieldCheckboxGroup
+                                    fieldLabel="Atom Types:"
+                                    fieldName="Atom Types"
+                                    fieldValue={[]}
+                                    checkValues={allAtomTypes.map((t)=>t.type)}/>
+                            </ManagedField>
+                        </div>
+                        <div className="atom-list-filters__page-size">
+                            <ManagedField data={this.state.params} updateData={this.updateAtomList} fieldLocation="page-size" name="Page size:">
+                                <FormFieldSelectBox selectValues={["20","50","100","150","200"]}/>
+                            </ManagedField>
+                        </div>
+                    </form>
+                </div>
+                <div className="atom-list">
+                    <table className="atom-list__table">
+                        <thead>
+                            <tr className="atom-list__header-row">
+                                <th className="atom-list__header">Title</th>
+                                <th className="atom-list__header">Type</th>
+                                <th className="atom-list__header">Last Modified</th>
+                                <th className="atom-list__header">Created by</th>
+                                <th className="atom-list__header">Publication status</th>
+                                <th className="atom-list__header"> </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                        {this.props.atomList.map((atom) => (
```

Comment:
Just for clarity... can this be pulled out into a seperate render method? e.g. `this.props.atomList.map(renderListItem)`